### DISCUSSION TITLE
About the use of Canta

### DISCUSSION THREAD

**Discussion by Dhyfer1** [samolego/Canta#10] [General]
Hi. I have followed the instructions. I have installed shizuku and started it via wireless debugging.

My question is: Should Shizuku be open and wireless debugging enabled every time I open Canta to uninstall an app?

Another question. Within Shizuku, there is an option to authorize apps, should I authorize Canta?

**Comment by samolego** [comment]
*This part I'm not 100% sure*;
No, you just need to have wireless debugging on when starting shizuku (it even turns off autonatically after disconnecting from wifi)

*This should be true though*:
Shizuku service needs to be running, yes. Note that after reboot you must restart it.

Yes, you need to grant shizuku access to canta. You can do this in shizuku app or canta will ask you for permission on first uninstall.
